javascriptv3/example_code/cross-services/transcription-app/src/helper.ts

```diff
@@ -13,7 +13,7 @@ Running the code:
 For more information, see https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
 
 */
-// This file contains vanilla JavaScript helper functions for user interface.
+// Vanilla JavaScript helper functions for user interface
```

Comment:
There needs to be a period at the end of the sentencde. Also it should say "for the user interface"